flowpipe/node.py

```diff
@@ -161,11 +161,35 @@ def evaluate(self):
         return outputs
 
     @abstractmethod
-    def compute(self, *args, **kwargs):
+    def compute(self, *args, **kwargs):  # pragma: no cover
         """Implement the data manipulation in the subclass.
 
         Return a dictionary with the outputs from this function.
         """
+        raise NotImplementedError("Compute must be overwritten")
+
+    def __rshift__(self, other):
+        """Syntactic sugar for connecting outputs of this node based on names.
+
+        If other is an _InPlug, connect the output with matching name.
+        If other is an INode, connect all outputs with matching names.
+        """
+        if isinstance(other, _InPlug):
+            try:
+                self.outputs[other.name].connect(other)
+            except KeyError:
+                raise KeyError("No output named {0}".format(other.name))
+        elif isinstance(other, INode):
+            no_connection = True
+            for key in self.outputs.keys():
+                if key in other.inputs:
+                    self.outputs[key].connect(other.inputs[key])
+                    no_connection = False
+            if no_connection:
+                raise ValueError("{0} has no matching inputs".format(
+                    other.name))
```

Comment:
I find it important to raise an Exception when no connection could be made, since the user obviously intended to connect something.

Reply:
I think we should have this in a `connect` method and then just call it in the rshift. THat way we keep the api between plugs and nodes identical

Reply:
I agree with the exception